### ISSUE TITLE
Example in README.md throws error

### ISSUE THREAD

**Issue by Reggino** [AppliedMathematicsANU/plexus-form#41]
Hi,

Cut and paste from the example in ```README.md``` gives the following error...

```
TypeError: Cannot read property 'find' of undefined
    at fullOrdering (plexus-form.js:923)
    at Object.types.object (plexus-form.js:904)
    at Object.module.exports [as make] (plexus-form.js:806)
    at module.exports.React.createClass.render (plexus-form.js:180)
    at ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext (ReactCompositeComponent.js:789)
```

Running ```react 0.13.3``` . I'll investigate further...

**Comment by bebraw** [comment]
Looks like the source relies on `props.schema['x-ordering']`. It blows up because `x-ordering` property is missing from the example schema. I think you can patch around it by setting `"x-ordering": []` at your schema.

Ideally this should be fixed on library level, though (just need to default an empty array there).

**Comment by odf** [comment]
I am slightly confused. Commit e98ff08, the latest from master, works for me (Firefox 38.0 on Linux), whereas the latest release does not, but produces a different error.

**Comment by bebraw** [comment]
@odf Looks like the current stable version and `master` version aren't exactly the same. There are a few changes in `master` that probably explain that.

**Comment by odf** [comment]
@bebraw Yes, it's probably time for a new release. I'm just confused because I don't see `fullOrdering`/`'x-ordering'` related errors in either version.
